### PR TITLE
Added new field for the ASN operator doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add the information about the newly added field value.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Documentation update](https://sumologic.atlassian.net/browse/DOCS-459)

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]

File: docs/search/search-query-language/search-operators/asn-lookup.md [hunk 1]
```diff
@@ -18,16 +18,15 @@ lookup\<field\> from asn://default on ip\<ip_address\>
...
 ## Example
 
-The following query references a data stream with IPv4 addresses, parses
-those IPv4 addresses, and then uses ASN Lookup to retrieve their
-autonomous system information. 
+The following query references a data stream with IPv4 addresses, parses those IPv4 addresses, and then uses ASN Lookup to retrieve their autonomous system information. 
 
 ```sql
-_sourceCategory=stream "remote_ip="
+_dataTier=all _sourceCategory=*/katta "remote_ip="
```
Comment (vikas-sumo) [inline docs/search/search-query-language/search-operators/asn-lookup.md:29]: lets not use `_dataTier=all _sourceCategory=*/katta`

use _sourceCategory=stream

Comment (krastogi-eng) [reply docs/search/search-query-language/search-operators/asn-lookup.md:29]: we should use some random sourceCategory instead of our internal assemblies

Comment (JV0812) [inline docs/search/search-query-language/search-operators/asn-lookup.md:29]:
```suggestion
_dataTier=all _sourceCategory=stream "remote_ip="
```

File: docs/search/search-query-language/search-operators/asn-lookup.md [hunk 2]
```diff
@@ -18,16 +18,15 @@ lookup\<field\> from asn://default on ip\<ip_address\>
 |:--|:--|
 | `*` | Use a wildcard (`*`) character as a shortcut to return both fields. |
 | `asn` | Autonomous System Number |
-| `organization` | Autonomous System Organization Name (Carrier) or ID in some cases. |
+| `organization` |  The name of the organization that owns the ASN or ID in some cases. This organization is responsible for the routing of traffic for network blocks. |
```
Comment (riotParrot) [inline docs/search/search-query-language/search-operators/asn-lookup.md:21]: can we keep `Carrier` within description for clarity

**Review by riotParrot — CHANGES_REQUESTED** [review]

File: docs/search/search-query-language/search-operators/asn-lookup.md (hunk 2 above)
Comment (JV0812) [inline docs/search/search-query-language/search-operators/asn-lookup.md:21]:
```suggestion
| `organization` |  The name of the organization that owns the ASN or ID in some cases. This organization (Carrier) is responsible for the routing of traffic for network blocks. |
```

**Pull request merged**